src/Jackett.Common/Services/ConfigurationService.cs

```diff
@@ -81,33 +81,44 @@ public void CreateOrMigrateSettings()
                     }
                     else
                     {
-                        PerformMigration();
+                        PerformMigration(oldDir);
                     }
-
                 }
                 catch (Exception e)
                 {
                     logger.Error($"ERROR could not migrate settings directory\n{e}");
                 }
             }
+
+            // Perform a migration in case of https://github.com/Jackett/Jackett/pull/11173#issuecomment-787520128
+            if (Environment.OSVersion.Platform == PlatformID.Unix)
+            {
+                var oldDirectory = "Jackett";
+                if (Directory.Exists(oldDirectory))
+                {
+                    PerformMigration(oldDirectory);
+                }
+            }
         }
 
-        public void PerformMigration()
+        public void PerformMigration(string oldDirectory)
         {
-            var oldDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Jackett");
-            if (Directory.Exists(oldDir))
+            if (Directory.Exists(oldDirectory))
```

Comment:
Invert this If to reduce nesting.

Reply:
Addressed in 1ade5585761a6e206580533ca99bed3d73d93625